### PR TITLE
Split yolo v8 optimization notebooks to 3 separate

### PR DESCRIPTION
**Pull request by sbalandi**
colab object detection - https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/yolov8/notebooks/230-yolov8-optimization/230-yolov8-object-detection.ipynb
instance segmentation - https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/yolov8/notebooks/230-yolov8-optimization/230-yolov8-instance-segmentation.ipynb
kypoint detection - https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/yolov8/notebooks/230-yolov8-optimization/230-yolov8-keypoint-detection.ipynb

Task:

CVS-111529

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

**Comment by raymondlo84** [comment]
Possible to have a simple demo app that runs all, but doesn't require to run all NNCF and all again?

**Review by itrushkin — CHANGES_REQUESTED** [review]

File: notebooks/230-yolov8-optimization/README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
-The tutorial consists of the following steps:
+- [230-yolov8-object-detection](./230-yolov8-object-detection.ipynb) [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-object-detection.ipynb)
+- [230-yolov8-instance-segmentation](./230-yolov8-instance-segmentation.ipynb) [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-instance-segmentation.ipynb)
+- [230-yolov8-keypoint-detection](./230-yolov8-keypoint-detection.ipynb) [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-keypoint-detection.ipynb)
+
+
+For each case, it will be describes the following steps in tutorials:
```
[the remark this answers is not in the document]
Comment (sbalandi) [reply notebooks/230-yolov8-optimization/README.md:19]: done

**Review by itrushkin — COMMENTED** [review]
I am unable to submit a review in ReviewNB App.
image

**Comment by sbalandi** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1281/discussion/#comment-1701012532'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sbalandi** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1281/discussion/#comment-1701044288'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sbalandi** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1281/discussion/#comment-1701044343'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by itrushkin — APPROVED** [review]

**Pull request merged**